packages/wallet/api/src/lib-wallet.js

```diff
@@ -1895,6 +1897,7 @@ export function makeWalletRoot({
   const importBankAssets = async bank => {
     observeIteration(subscribeEach(E(bank).getAssetSubscription()), {
```

Comment:
```suggestion
    observeIteration(subscribeEach(E(bank).getAssetSubscription()), harden({
```
or
```suggestion
    observeIteration(subscribeEach(E(bank).getAssetSubscription()), Far(..., {
```
Nothing to do with this PR, but a good drive-by fix. I noticed this because its absence made the code harder to visually parse, which is interesting. In context, on first visual pass, I mistook this open curly to be a function-body open curly rather than an object-literal open curly.